test/unit/test_formula.py

```diff
@@ -279,6 +287,56 @@ def test_generate_formula_no_test():
     assert 'test do' not in formula
 
 
+def test_generate_formula_multiline_test():
+    """Tests that we generate the formula content correctly (when there is no test).
+
+    NOTE: See docstring in `_record_formula` for more details on how recording formulas works.
+    """
+    formula_filename = f'{inspect.stack()[0][3]}.rb'
+    mock_repo_name = formula_filename.replace('_', '-').replace('.rb', '')
+    mock_tar_url = f'https://github.com/{USERNAME}/{mock_repo_name}/archive/refs/tags/v0.1.0.tar.gz'
+
+    repository = {
+        'description': DESCRIPTION,
+        'license': LICENSE,
+    }
+
+    formula = Formula.generate_formula_data(
+        owner=USERNAME,
+        repo_name=mock_repo_name,
+        repository=repository,
+        checksums=[
+            {
+                f'{mock_repo_name}.tar.gz': {
+                    'checksum': CHECKSUM,
+                    'url': (
+                        f'https://github.com/justintime50/{mock_repo_name}/releases/download/{VERSION}/{mock_repo_name}-{VERSION}.tar.gz'  # noqa
+                    ),
+                },
+            }
+        ],
+        install=INSTALL,
+        tar_url=mock_tar_url,
+        depends_on=DEPENDS_ON,
```

Comment:
question: To protect against regressions, can we test all fields that should be multiline here? Meaning can we have a multi-line `install` and `formula_includes` as well?